fortinet_fortigate/icon_fortinet_fortigate/connection/connection.py

```diff
@@ -60,6 +64,31 @@ def get_address_group(self, address_group_name):
         group = groups[0]
         return group
 
+    def get_address_object(self, address_name):
+        response_ipv4_json = self.session.get(
```

Comment:
The responses in this function need to be wrapped in try/except and reasonable error messages returned. 

So like 
```
try: 
response_ipv4_json.raise_for_status()
except Exception: 
if response_ipv4_json.status_code = 400
...
```

Reply:
@jmcadams-r7 could you let me know why try/catch is required? I've tested this code with different respons codes from Fortigate API and it works fine. The reason I've used `if` rather then `try/catch` is mainly for readability. Since, we want to try and find IPv4 address and in case it does not exists, try to find IPv6, we would have to use inner try/catch?